packages/flutter_tools/test/general.shard/build_system/targets/web_test.dart

```diff
@@ -964,7 +990,7 @@ void main() {
       JsCompilerConfig(optimizationLevel: 0),
       JsCompilerConfig(noFrequencyBasedMinification: true),
       JsCompilerConfig(sourceMaps: false),
-      JsCompilerConfig(renderer: WebRendererMode.canvaskit),
+      JsCompilerConfig(renderer: WebRendererMode.skwasm),
```

Comment:
We should do `WebRendererMode.auto` or `WebRendererMode.html` here for this test, I think. `JsCompilerConfig` with a skwasm renderer mode, while it won't throw at this particular layer it isn't a logically valid or supported combination, so it's weird to use it in the test.

Reply:
The only reason I used skwasm instead of html is because I didn't want to change it again when we remove the html renderer.

But I see what you're saying. I'll use html for now, and later remove this line entirely when there's no more html renderer.